hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/pipeline/WritableECContainerProvider.java

```diff
@@ -128,6 +129,8 @@ public ContainerInfo getContainer(final long size,
       synchronized (pipeline.getId()) {
         try {
           ContainerInfo containerInfo = getContainerFromPipeline(pipeline);
+          Optional.ofNullable(containerInfo)
+              .ifPresent(ContainerInfo::updateLastUsedTime);
```

Comment:
At this point in the code, we are not sure if this container is going to be used or not, as it may be excluded by the logic just below this.

I think the update needs to be just before the return at line 143 in the diff:

```
} else {
  containerInfo.updateLastUsedTime();
  return containerInfo;
}
```   

Does the lastUsedTime get set correct in the allocateContainer() call when the container is first created?